File: FizzBuzzBizz.php
<?php
for($i=1; $i<=100; $i++){

    if( $i % 35 == 0 ){
        echo "BuzzBizz" . "\r\n";
    }else if( $i % 15 == 0 ){
        echo "FizzBuzz" . "\r\n";
    }else if( $i % 7 == 0 ){
        echo "Bizz" . "\r\n";
    }else if( $i % 5 == 0 ){
        echo "Buzz" . "\r\n";

    }else if( $i % 3 == 0 ){
        echo "Fizz" . "\r\n";

    }else{
        echo $i . "\r\n";
    }

}

File: fizzbuzzbizz.rs
fn main() {
    for i in 1..100 {
        if i % 35 == 0 {
            println!("BuzzBizz");
        } else if i % 15 == 0 {
            println!("FizzBuzz");
        } else if i % 7 == 0 {
            println!("Bizz");
        } else if i % 5 == 0 {
            println!("Buzz");
        } else if i % 3 == 0 {
            println!("Fizz");
        } else {
            println!("{}", i);
        }
    }
}
